content/webapp/views/pages/concepts/concept/concept.styles.tsx

```diff
@@ -124,7 +124,8 @@ export const ConceptImages = styled(Space).attrs({
 })`
   background-color: ${props => props.theme.color('black')};
 
-  h2 {
+  /* The class is necessary because the image modals styles get overwritten otherwise */
+  h2.sectionTitle {
```

Comment:
does it still need to be qualified by the `h2` (i.e. could it just be `.sectionTitle`)?